### PR TITLE
Feature/hidable display labels

### PR DESCRIPTION
**Pull request by jackmcdade**
References statamic/ideas#262

### CODE REVIEW

File: resources/js/components/fieldtypes/IntegerFieldtype.vue [hunk 1]
```diff
@@ -19,7 +19,5 @@ export default {
 
     mixins: [Fieldtype],
 
-    props: ['focus']
-
 };
 </script>
```
Comment (jasonvarga) [inline resources/js/components/fieldtypes/IntegerFieldtype.vue:23]: Are these lines relevant to this PR?

File: resources/js/components/DropdownList.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@
                 </button>
             </slot>
         </template>
-        <slot />
+        <template #default><slot /></template>
```
Comment (jasonvarga) [inline resources/js/components/DropdownList.vue:10]: Is this relevant to this PR?

File: resources/js/components/fieldtypes/IntegerFieldtype.vue (hunk 1 above)
Comment (jasonvarga) [reply resources/js/components/fieldtypes/IntegerFieldtype.vue]: Nevermind, I merged master back into this and it's gone. 👍

File: resources/js/components/DropdownList.vue (hunk 2 above)
Comment (jasonvarga) [reply resources/js/components/DropdownList.vue]: Nevermind, I merged master back into this and it's gone. 👍

**Comment by jasonvarga** [comment]
Nice!

A few things to think about:

1. The UI is a bit cramped especially at narrower breakpoints.
   
   image

2. Should we make it `hide_label` instead and it defaults to a disabled toggle? That way you don't end up with `show_label: true` on 99% of your blueprint fields. If we flip it, any `hide_label: false`s would get stripped out.

3. Or really... should it be called `hide_form_label: true` to be explicit that we're talking about it in forms and not in listings? I'm thinking about if we introduce a feature like https://github.com/statamic/ideas/issues/51 (which would probably be something like `listing_display`).

**Review by jasonvarga — COMMENTED** [review]
Nice!

A few things to think about:

1. The UI is a bit cramped especially at narrower breakpoints.
   
   image

2. Should we make it `hide_label` instead and it defaults to a disabled toggle? That way you don't end up with `show_label: true` on 99% of your blueprint fields. If we flip it, any `hide_label: false`s would get stripped out.

3. Or really... should it be called `hide_form_label: true` to be explicit that we're talking about it in forms and not in listings? I'm thinking about if we introduce a feature like https://github.com/statamic/ideas/issues/51 (which would probably be something like `listing_display`).

**Comment by jackmcdade** [comment]
@jasonvarga All good points – i went back and forth a few times on that too and am always torn on consistency between "enable to disable" or "enabled but saving data". 🤔 You're probably right here though.

This is what I **really** wanted to do tho:

image


Would need to be a custom system "label" fieldtype that set two keys on the array. So i didn't tackle it.

**Comment by jasonvarga** [comment]
Let's circle back on this one. There's merge conflicts and other things to work out.
